lightning/src/offers/invoice.rs

```diff
@@ -762,13 +762,37 @@ macro_rules! invoice_accessors { ($self: ident, $contents: expr) => {
 	}
 } }
 
+
+macro_rules! invoice_accessors_signing_pubkey {
```

Comment:
We have separated the macros for invoice, and static invoice. However, the macro name, structure, docs, and variable names are identical. Maybe we can update their docs the have information specific to each type?

Reply:
These macros are private to the module, so I think it is fine to keep them as is.